Artifacts.toml

```diff
@@ -0,0 +1,39 @@
+[densenet121]
```

Comment:
Which script was used to generate these?

Reply:
https://github.com/FluxML/MetalheadWeights contains the scripts